config-freebsd.md

```diff
@@ -0,0 +1,124 @@
+# <a name="FreeBSDContainerConfiguration" />FreeBSD Container Configuration
+
+This document describes the schema for the [FreeBSD-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+
+## <a name="configFreeBSDDevices" />Devices
+
+Devices in FreeBSD are accessed via the `devfs` filesystem. Typically, each container will have a `devfs` filesystem mounted into it's `/dev` directory. Often, a minimal set of devices is exposed to the container using ruleset 4 from `/etc/defaults/devfs.rules` - the ruleset is specified as a mount option.
+
+Optionally, additional devices can be exposed to the container using an array of entries inside the `devices` root field:
+
+* **`path`** _(string, REQUIRED)_ - the device path relative to `/dev`
+* **`mode`** _(integer, OPTIONAL)_ - device permissions as an integer which is interpreted as in `chmod(1)`.
+
+### Example
+```json
+"devices": [
+	{
+        "path": "pf",
+        "mode": "0700"
+    }
+]
+```
+
+## <a name="configFreeBSDJail" />Jail
```

Comment:
This section feels to me more like documentation for an existing implementation than specification from which a new compatible implementation could be implemented. 🤔

Are there specific features of FreeBSD that could be explicitly named here to help connect these things more strongly?  Are there any places here where SHOULD or MUST language would be appropriate?

Reply:
This section is informational and is intended as a guide for readers familiar with FreeBSD jails to help them understand the mapping from the FreeBSD OCI config settings to the corresponding jail.

Since this proposal only covers the use of FreeBSD jails to implement OCI containers, there is very little freedom for implementors. The text following the table would benefit from using SHOULD/MUST language.